frontend/src/lib/components/PathCleanFilters/PathCleanFilterItem.tsx

```diff
@@ -25,7 +25,7 @@ export function PathCleanFilterItem({ filter, onChange, onRemove }: PathCleanFil
 
     return (
         <>
-            {visible && (
+            {visible && onChange && (
```

Comment:
did you mean to include this? I'm pretty low on context on this PR but this looks unrelated

Reply:
I did. This component is now used on the describer and I dont want the hover state/action